pkg/tasks/machine_deployments.go

```diff
@@ -34,7 +34,13 @@ func createMachineDeployments(s *state.State) error {
 		return nil
 	}
 
+	if !s.CreateMachineDeployments {
+		s.Logger.Info("Skipped creating MachineDeployments...")
```

Comment:
Super-tiny nit: I would use a single dot instead of three dots because we don't wait for anything here.

```suggestion
		s.Logger.Info("Skipped creating MachineDeployments.")
```

Reply:
:heavy_check_mark: